curl/Dockerfile

```diff
@@ -36,7 +40,7 @@ ARG SIG_ALG
 ARG DEFAULT_GROUPS
 ARG MAKE_DEFINES
 
-LABEL version="4"
+LABEL version="5"
```

Comment:
The opencontainers spec defines a list of standard annotations which could be used to add in important metadata. This superceeds the earlier label schema

See https://github.com/opencontainers/image-spec/blob/main/annotations.md

For instance this might be
```
LABEL org.opencontainers.image.version="5"
```

It may be worth opening a new issue just on this specific aspect

Reply:
opened https://github.com/open-quantum-safe/oqs-demos/issues/300

Reply:
I agree with the idea, lets let some discussion occur on that issue first and handle it in a separate PR later though